espnet2/asr/frontend/default.py

```diff
@@ -9,8 +9,10 @@
 from torch_complex.tensor import ComplexTensor
 from typeguard import check_argument_types
 
-from espnet.nets.pytorch_backend.frontends.frontend import Frontend
 from espnet2.asr.frontend.abs_frontend import AbsFrontend
+
+# from espnet.nets.pytorch_backend.frontends.frontend import Frontend
+from espnet2.enh.nets.beamformer_net import BeamformerNet as Frontend
```

Comment:
Sorry, I changed my mind. Could you restore to initial state?
I think this PR shouldn't touch asr/frontend.

Reply:
OK, I can do it.

Reply:
Done.